.editorconfig

```diff
@@ -0,0 +1,44 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[{*.enc,*.go,*.gpg,*.md,*.svg}]
+trim_trailing_whitespace = false
+
+[{*.enc,*.gpg,*.json,*.svg,LICENSE}]
+insert_final_newline = unset
+
+[{*.enc,*.gpg,*.svg}]
+end_of_line = unset
+
+[{*.enc,*.go,*.gpg,*.svg,*.html,LICENSE}]
+indent_size = unset
+indent_style = unset
+
+[{*.enc,*.gpg,*.json,*.svg,LICENSE}]
+max_line_length = unset
+
+[*.{go,js}]
+# Allow C-style doc comments
+block_comment_start = /*
+block_comment = *
+block_comment_end = */
+
+[{*.make,go.mod,go.sum,Makefile,makefile}]
```

Comment:
We don't have lowercase `makefile` in here after you address @johanstokking's comment.